firmware/boron-ins-fsm/test/mocks/mock_ble.h

```diff
@@ -33,16 +33,15 @@ class BleAdvertisingData {
 };
 
 // Fake class for BleScanResult object
-class BleScanResult
-{
+class BleScanResult {
 private:
     BleAddress address_;
     BleAdvertisingData advertisingData_;
     BleAdvertisingData scanResponse_;
     int8_t rssi_;
 
 public:
-    const BleAddress address() const {;
+    const BleAddress address() const {
         return address_;
     }
```

Comment:
Any idea what was happening here with the semi-colon? Was that necessary for anything?

Reply:
I can only imagine that this was a typo, which happened to be placed somewhere where it didn't actually affect anything.